spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -96,6 +95,8 @@
 
 	private Executor asyncPullExecutor = Runnable::run;
 
+	private Map<String, SubscriberStub> collectionOfSubscriberStub = new HashMap<>();
```

Comment:
naming suggestion: `subscriptionToSubscriberStubMap`.

Reply:
Thank you! Done.